### PR TITLE
[Student][E2E][MBL-12978] | E2E tests for assignments

### PR DESCRIPTION
**Pull request by joehoag**

### CODE REVIEW

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentListPage.kt [hunk 1]
```diff
@@ -43,8 +56,45 @@ class AssignmentListPage : BasePage(pageResId = R.id.assignmentListPage) {
...
+        else {
+            val matcher = allOf(
+                    withText(R.string.toDoNoDueDate),
+                    withId(R.id.date),
+                    withParent(withParent(withChild(withText(assignment.name)))))
+            scrollToAndAssertDisplayed(matcher)
+        }
+
+        // Check that grade is present, if that is specified
+        //println("expectedGrade=$expectedGrade")
```
Comment (StephenBrough) [inline apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentListPage.kt]: Minor nitpick, looks like a deprecated comment

Comment (joehoag) [reply apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentListPage.kt]: The comment is valid.  But I can remove the commented-out println() statement.

Comment (StephenBrough) [reply apps/student/src/androidTest/java/com/instructure/student/ui/pages/AssignmentListPage.kt]: Yeah, sorry, that's what I meant :)

**Comment by joehoag** [comment]
Thanks for the review!

**Pull request merged**